src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/VsProject.cs

```diff
@@ -30,17 +30,20 @@ public partial class VSProject : VSLangProj.VSProject, IConnectionPointContainer
         private readonly VSLangProj.VSProject _vsProject;
         private readonly IProjectThreadingService _threadingService;
         private readonly ActiveConfiguredProject<ProjectProperties> _projectProperties;
+        private readonly BuildManager _buildManager;
 
         [ImportingConstructor]
         internal VSProject(
             [Import(ExportContractNames.VsTypes.CpsVSProject)] VSLangProj.VSProject vsProject,
             IProjectThreadingService threadingService,
             ActiveConfiguredProject<ProjectProperties> projectProperties,
-            UnconfiguredProject project)
+            UnconfiguredProject project,
+            BuildManager buildManager)
```

Comment:
This import is in conflict with the export; 

- import lives in component that is _C# | VB_. 
- export is _(C# | VB) & LanguageService_

Is CPS exporting "BuildManager" at all?

Reply:
Discussed offline, solution is as above.